src/internal/connector/onedrive/collection.go

```diff
@@ -506,7 +506,36 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 				item)
 
 			if err != nil {
-				el.AddRecoverable(clues.Wrap(err, "getting item metadata").Label(fault.LabelForceNoBackupCreation))
+				// Skip if the item was deleted by the time we attempted to get
```

Comment:
@meain Quick question: Is the `oc.itemMetaReader` call expected for files only? Line 501 above.

Reply:
The comment is misleading, it happens for both files and folders. In case of folders, we will only have the metadata and no data.